tgui/packages/tgui/interfaces/CommunicationsConsole.js

```diff
@@ -12,29 +12,39 @@ const STATE_MESSAGES = "messages";
 // Used for whether or not you need to swipe to confirm an alert level change
 const SWIPE_NEEDED = "SWIPE_NEEDED";
 
+const sortByCreditCost = sortBy((a, b) => a.creditCost - b.creditCost);
```

Comment:
Iteratee function is expected to return the value by which it sorts, not -1, 0 or 1, and b argument might not even exist.

```
sortBy(item => item.creditCost)
```